app/templates/games/list.hbs

```diff
@@ -0,0 +1,17 @@
+{{!-- Only allow game creation if you are not anonymous. --}}
+{{#unless session.user.isAnonymous}}
```

Comment:
not sure we should replicate the deck validation logic here as it may be hard to explain why this button doesn't show up to a new user